docs/concepts/overview/working-with-objects/common-labels-annotations.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Common Label and Annotation Recommendations
+---
+
+{% capture overview %}
+Many tools can be used to visualize and manage Kubernetes objects. This goes beyond
+tools such as dashboard or kubectl and includes many tools in the Kubernetes ecosystem.
```

Comment:
“Many tools” is repeated twice in two sentences - seems redundant. Suggested wording change:

> The tools used to manage objects goes beyond kubectl and the dashboard.